virttest/utils_libvirt/libvirt_pcicontr.py

```diff
@@ -51,3 +54,30 @@ def get_free_pci_slot(vm_xml, max_slot=31):
         if slot not in used_slot:
             return slot
     return None
+
+
+def reset_pci_num(vm_name, num=15):
+    """
+    Reset the number of guest pci, add 15 by default
+
+    :param vm_name: VM name
+    :param num: The number of expected pci
+    """
+    vmxml = vm_xml.VMXML.new_from_dumpxml(vm_name)
+    # This func only works on aarch64 and x86/q35 machine
+    if 'aarch64' in vmxml.os.arch \
+            or 'q35' in vmxml.os.machine:
+        # Default pcie setting
+        pcie_root_port = {'controller_model': 'pcie-root-port', 'controller_type': 'pci'}
+        ret_indexes = get_max_contr_indexes(vmxml, 'pci', 'pcie-root-port')
+        cur_pci_num = int(ret_indexes[0])
+        logging.debug("cur_pci_num is %d", cur_pci_num)
```

Comment:
The logging message could be much formal, like 'The current maximum PCI controller index is: ...'

Reply:
Ok, got it. Thanks a lot !